src/task_5_3.py

```diff
@@ -0,0 +1,14 @@
+# 3)Два натуральных числа называют дружественными,если каждое из них
+# равно сумме всех делителей другого, кроме самого этого числа. Найти все
+# пары дружественных чисел, лежащих в диапазоне от 200 до 300. [02-3.2-HL02]
+for i in range(200, 301):
+    k = 0
+    for x in range(1, i):
```

Comment:
Чтобы найти делитель числа, не нужно проходиться по всем числам. Достаточно только половины ```suggestion
    for x in range(1, i // 2):
```